test/e2e/autoscaling_test.go

```diff
@@ -21,14 +21,14 @@ import (
 
 func TestAutoscaling(t *testing.T) {
 	t.Parallel()
-	g := NewWithT(t)
 
 	ctx, cancel := context.WithCancel(testContext)
 	defer cancel()
 
 	clusterOpts := globalOpts.DefaultClusterOptions(t)
 
 	e2eutil.NewHypershiftTest(t, ctx, func(t *testing.T, mgtClient crclient.Client, hostedCluster *hyperv1.HostedCluster) {
+		g := NewWithT(t)
```

Comment:
```
or should this be in the hypershiftTestFunc signature and passed through in the Execute() function
	if h.test != nil && !h.Failed() {
		h.Run("Main", func(t *testing.T) {
			h.test(t, **g**, h.client, hostedCluster)
		})
	}
```

so it's not possible for a test to use the one with wider scope?

Reply:
nice idea!